driver.py

```diff
@@ -26,15 +26,9 @@ def parse_options():
     parser.add_argument('--app', choices=['iotjs', 'jerryscript'], default='iotjs',
                         help='the target application (default: %(default)s)')
 
-    parser.add_argument('--branch', metavar='name', default='master',
```

Comment:
IMHO, the `branch` and `commit` options are useful, even they are not used now.

Reply:
If you will use the `js-remote-test` as a submodule of `iotjs`, I think the user should change the branch and commit manually that he or she want to test. Furthermore, if the `iotjs` had any modifications, you could not use these options, because `git` does not allow to change branch or commit (e.g. because the actual branch is not clean).

Not, of course we can use these options because `iotjs` is a part of `js-remote-test` and assumed that `iotjs` does not have any modifications.

Reply:
@robertsipka What is your opinion about this?

Reply:
In fact, it is not fully supported, because for the memory measurement we apply patches beside NuttX with IoT.js on stm32f4-discovery board. They are regularly updated and following the master branch. It may easily results several errors on other branches or commits. I think we should remove it, so the developer can decide to change into another branch or commit, and run the remote-tester, but in this case it may needed some changes in the patches.

Reply:
Thanks for the explanation, I can accept this, but the current solution `utils.execute(paths.<app_path>, 'git', ['pull', 'origin', 'master'])` also can lead builderrors, if certain files modified locally.

Reply:
Yes, you are right. That should be also removed.